### PR TITLE
Add indicators to admin panel

### PR DESCRIPTION
**Pull request by nick-beyondessential**
https://github.com/beyondessential/tupaia-backlog/issues/1255

Exposing Indicators via Admin Panel

In This PR:
- branched from epic-984-meditrak-server-permissions (original PR for dev https://github.com/beyondessential/tupaia/pull/1381)
- Add an Indicators tab under Visualisations in admin panel
- Includes list, add, edit
- Add meditrak endpoints

### CODE REVIEW

**Comment by edmofro** [comment]
Nice one @nick-beyondessential! Unfortunately, and this is completely my fault for not mentioning it earlier, but we've had to rework the way we handle GET endpoints 😬 

@EMcQ-BES realised the filtering needs to be done inside SQL, and that the unit tests should be done at the endpoint handler rather than asserter level, so there'll be a bit to do here. I'm so sorry about that - I should have communicated better!

Please talk to Ethan about what's involved

**Comment by edmofro** [comment]
@billli0 why did you close this?

**Comment by billli0** [comment]
I made a mistake to delete branch `epic-984-meditrak-server-permissions`. Thanks to Ethan, he had local copy and restored the branch. I'm very sorry to bring trouble to you guys.

**Comment by edmofro** [comment]
@EMcQ-BES is this ready for review?

**Comment by EMcQ-BES** [comment]
@edmofro 

Yes? Just a few notes:
- I swapped the permissions check to just check `BES Admin`, I figured indicators are closely tied to data elements which are BES only so it made sense.
- The prior permissions check is still there in case you want me to revert that decision, otherwise it's dead code
- I haven't updated the unit tests, I've just `xdescribe`-d them for now

File: packages/meditrak-server/src/routes/index.js [hunk 1]
```diff
@@ -149,6 +153,7 @@ export default {
   getDisasters: useRouteHandler(GETDisasters),
   getDashboardReports: useRouteHandler(GETDashboardReports),
   getDashboardGroups: useRouteHandler(GETDashboardGroups),
+  getIndicators: useRouteHandler(GETIndicators),
```
Comment (edmofro) [inline packages/meditrak-server/src/routes/index.js:156]: Could make a `BESAdminGetHandler` for consistency, even if it's the only one to use it

**Review by edmofro — CHANGES_REQUESTED** [review]
Agree with your decision to go with BES Admin (for now at least)

Please:
- delete dead permissions check
- update unit tests if it'll take < 2 hours, and delete otherwise
- get this out of draft and onto the reviews channel

File: packages/meditrak-server/src/routes/index.js (hunk 1 above)
Comment (EMcQ-BES) [reply packages/meditrak-server/src/routes/index.js]: Apparently I already wrote one but forgot about it

**Review by edmofro — APPROVED** [review]

**Pull request merged**